pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -84,7 +84,7 @@ typedef struct {
     __u64 request_started;
 
     __u16 response_status_code;
-    __u8 request_method;
+    __u32 request_method;
```

Comment:
why is it required?

Reply:
The request value is compared with static table values, which is now an u32 with the removal of the packed attribute. You can't compare u8 and u32 in Golang.

Reply:
please open a jira ticket on that issue, we should fix it in a separate PR

Reply:
Done. The ticket is USMO-413.